public/env.json

```diff
@@ -1,6 +1,7 @@
 {
-  "BLOCKCHAIN_URL": "wss://ws.framenode-2.r0.dev.sora2.soramitsu.co.jp",
-  "SUBQUERY_ENDPOINT": "https://api.subquery.network/sq/sora-xor/sora-dev",
+  "BLOCKCHAIN_URL": "wss://mof2.sora.org",
```

Comment:
BLOCKCHAIN_URL - please revert this line

Reply:
Reverted